src/styles/1_base/_base.css

```diff
@@ -0,0 +1,15 @@
+@layer base {
+    body {
+        @apply tw-antialiased tw-font-open-sans tw-text-base;
+    }
+    h1 { @apply tw-text-4xl; }
```

Comment:
niiice, are these responsive as the viewport scales?

Reply:
Not yet, but we can add different classes to adjust the size in the html, like `<h1 class="sm:tw-text-xl">`. I'm hoping to add this to the Styelsora but I would like to get the mobile specs first. I based the sizes on Mitch's design doc and found the closest matching tailwind sizes.